GSA-TTS/FAC
# Sk/extra cap references

**Closes**: https://github.com/GSA-TTS/FAC/issues/3709, https://github.com/GSA-TTS/FAC/issues/3884

This PR handles invalid audits that have no findings / less findings in comparison to the captexts.  These audits are flagged and allowed to migrate as-is.

**Testing**:

1. Test with a failed audit: (dbkey = 105561, audit_year = 2019) if access to census data in Preview is available.

- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that dissemination_captext contains the captexts from census eleccaptext.
- Verify that dissemination_finding contains no findings.
- Verify that cap_text content is in the invalid audit record in dissemination_invalidauditrecord table.

2. Test by removing findings for an existing audit.
- For an audit that has captexts in the census eleccaptext table, 

1. remove findings in the census elecauditfindings table.
2. set FINDINGSCOUNT in elecaudits table to 0.
3. remove finding texts in the elecfindingstext table.

- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that dissemination_captext contains the captexts from census eleccaptext.
- Verify that dissemination_finding contains no findings.
- Verify that cap_text content is in the invalid audit record in dissemination_invalidauditrecord table.


## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
